### PR TITLE
fix(pagerduty): Use cURL instead of PagerDuty agent to send alarms.

### PR DESCRIPTION
**Pull request by elisiariocouto**
##### Summary
Currently, netdata needs the PagerDuty Agent to send alarms. We can use Events API to send the alarms with cURL.

This replaces #2976 and #4673 since the original author is not active on GitHub.

##### Component Name

PagerDuty integration

##### Additional Information

Uses `curl` instead of `pd-send` to send alarms to PagerDuty using Events API v1.
Cleanup whitespaces in  `alarm-notify.sh.in`.

### CODE REVIEW

File: health/notifications/alarm-notify.sh.in [hunk 1]
```diff
@@ -1871,7 +1864,7 @@ send_awssns() {
             info "sent Amazon SNS notification for: ${host} ${chart}.${name} is ${status} to '${target}'"
             sent=$((sent + 1))
         else
-            error "failed to send Amazon SNS notification for: ${host} ${chart}.${name} is ${status} to '${target}'" 
+            error "failed to send Amazon SNS notification for: ${host} ${chart}.${name} is ${status} to '${target}'"
```
Comment (Ferroin) [inline health/notifications/alarm-notify.sh.in]: This type of cleanup is probably a good thing, but it is unrelated to the PR.  At minimum, it should ideally be an independent commit.

Comment (elisiariocouto) [reply health/notifications/alarm-notify.sh.in]: I understand. The whitespace removal was done automatically by my editor. Do you want me to try to revert these changes?

Comment (Ferroin) [reply health/notifications/alarm-notify.sh.in]: I'm not sure, lets see what @ktsaou says about it, he may just not care.

Comment (paulfantom) [reply health/notifications/alarm-notify.sh.in]: Personally I don't care :) If it was some larger PR with many such lines changed then I would care, but for one line - nah.

Comment (elisiariocouto) [reply health/notifications/alarm-notify.sh.in]: There are actually some changes regarding whitespace, but I think it is pretty much straightforward to review these. But let me know.

**Comment by ktsaou** [comment]
So, is this ready to be merged?

**Comment by Ferroin** [comment]
I think so, I just forgot to actually do so.
